buildchain/buildchain/iso.py

```diff
@@ -107,7 +107,53 @@
         destination_directory=constants.ISO_ROOT,
         source_prefix=Path('scripts'),
         task_dep=['_iso_mkdir_root']
-    )
+    ),
+    helper.FileTree(
+        basename='_iso_add_tree',
+        files=(
+            Path('metalk8s'),
+
+            Path('metalk8s_cli/__init__.py'),
+            Path('metalk8s_cli/__main__.py'),
+            helper.TemplateFile(
```

Comment:
Not sure we need this, can read it (at runtime) from `product.txt`, since this is not a redistributable/system-wide-installable package

Reply:
OK, will do

Reply:
Discussion concluded that we will not include `kubectl` plugin(s) in this package, as they would probably be written in Go to make use of the available libraries.